src/options.rs

```diff
@@ -772,6 +777,46 @@ fn get_retention(matches: &ArgMatches, config: &Config) -> error::Result<u64> {
     }
 }
 
+fn get_network_legend(
+    matches: &ArgMatches, config: &Config,
+) -> error::Result<Option<LegendPosition>> {
+    let error = BottomError::ConfigError("cannot set a legend position. It have these values ['top', 'topleft', 'topright', 'left', 'right', 'bottom', 'bottomright', 'bottomleft', 'none'".to_string());
```

Comment:
```suggestion
    let error = BottomError::ConfigError("cannot set the network legend position, it must be one of: ['top', 'topleft', 'topright', 'left', 'right', 'bottom', 'bottomright', 'bottomleft', 'none'".to_string());
```

Reply:
Also it would probably make more sense to declare this as a closure and pass the closure when running `.map_err` (same for memory).

e.g.:

```suggestion
    let error = |_| BottomError::ConfigError("cannot set a legend position. It have these values ['top', 'topleft', 'topright', 'left', 'right', 'bottom', 'bottomright', 'bottomleft', 'none'".to_string());
```